locale/ja/foreman_bootdisk.po

```diff
@@ -77,28 +78,28 @@ msgid "Download subnet generic image"
 msgstr "サブネットの汎用イメージのダウンロード"
 
 msgid "Ensure %{file} is readable (or update \"Grub2 directory\" setting)"
-msgstr "%{file} が読み取り可能であることを確認します (または「Grub2 ディレクトリー」設定を更新します)"
+msgstr "%%{file} が読み取り可能であることを確認します (または「Grub2 ディレクトリー」設定を更新します)"
```

Comment:
The `%%{TAG}` appears to be a repeating problem. I think I've also seen this in Transifex, so it isn't some pulling problem.